calc/states_for_test.go

```diff
@@ -1027,6 +1054,10 @@ var vxlanTunnelIPDelete = vxlanWithBlock.withKVUpdates(
 	KVPair{Key: remoteHostVXLANTunnelConfigKey, Value: nil},
 ).withName("VXLAN host tunnel IP removed").withRoutes().withVTEPs()
 
+var vxlanMACDelete = vxlanWithBlockAndMAC.withKVUpdates(
+	KVPair{Key: remoteHostVXLANTunnelMACConfigKey, Value: nil},
+).withName("VXLAN host tunnel MAC removed").withRoutes().withVTEPs()
```

Comment:
After deleting the MAC, it should go back to using the auto-generated MAC so there should be routes and VTEPs.

The calc graph's invariant is that if you send updates A and B and then send a delete for B then the resulting state should go back to A.

Reply:
Acknowledged and changed.